cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -142,6 +143,7 @@ class EvaluatorCommand : OrtCommand(
                 "contain a single package configuration."
     ).convert { it.expandTilde() }
         .file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
+        .default(ortConfigDirectory.resolve(ORT_PACKAGE_CONFIGURATIONS_DIRNAME))
```

Comment:
commit-msg: "This aligns with the behavior for package curations." could be mis-leading, because in evaluator command the `--package-curations-dir` option does not default to any dir.

Would it make sense to just drop that sentence?

Reply:
> commit-msg: "This aligns with the behavior for package curations." could be mis-leading, because in evaluator command the `--package-curations-dir` option does not default to any dir.

Yeah, that's the special case of overriding.

> Would it make sense to just drop that sentence?

I'd instead prefer to write "... for package curations in the analyzer."

Agreed?